package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "schema-dot-org-markup",
   "version": "0.1.0",
```

Comment:
This version is irrelevant, as this root package.json is never published. For the same reason, the name here is irrelevant, no `main` is required, and this file should definitely have `"private": true,` set.